addons/redis/templates/componentdefinition-redis-sentinel.yaml

```diff
@@ -102,6 +119,8 @@ spec:
       env:
         - name: SENTINEL_SERVICE_PORT
           value: "26379"
+        - name: REDIS_SENTINEL_USER
+          value: "kbreplicator-sentinel"
```

Comment:
where this account will be used for?

Reply:
here `echo "sentinel auth-user $master_name $REDIS_SENTINEL_USER" >> /data/sentinel/redis-sentinel.conf`

Reply:
> where this account will be used for?

this is the username of redis for redis-sentinel connection